distribution/lib/Standard/Table/0.0.0-dev/src/Table.enso

```diff
@@ -226,7 +226,7 @@ type Table
         ? Consistent Row Count
         All columns must have the same number of rows. If a column has a
         different number of entries, an error is thrown.
-    input (columns : Vector (Vector | Column) = [['Column 1', []]]) -> Table =
+    input (columns : Vector (Vector | Column)) -> Table =
```

Comment:
Perhaps to avoid breaking the `Fixed_Width_Layout` when it gets an empty `Table.input`, we could add the following conversion:
```
Fixed_Width_Layout.from (that : Function) =
    is_unapplied_table_input = (that.to_text.starts_with "Table.type.new") || (that.to_text.starts_with "Table.type.input")
    if is_unapplied_table_input then
        Error.throw (Illegal_Argument.Error "Please ensure that the table specifying the format has 2 or 3 columns.")
    Panic.throw (Type_Error.Error Fixed_Width_Layout that "Expected to get a Fixed_Width_Layout or a Table describing it.")
```

This will intercept the unapplied `Table.input` with no input and replace it with a better error (also did same for `Table.new` as IMO worth to have too).

Reply:
![image](https://github.com/user-attachments/assets/3d868e12-ec95-45a2-8b32-8232bd42a43e)

Applied.